.github/CODEOWNERS

```diff
@@ -1,11 +1,11 @@
 # We are limiting this to 4 engineers and choosing not to select "Require review from Code Owners."
-* @acouch @aplybeah @chouinar @SammySteiner
+* @acouch @chouinar @mdragon 
 
-/documentation/ @sumiat @widal001 @andycochran @acouch @SammySteiner
-/frontend/ @andycochran @acouch @SammySteiner
-/api/ @jamesbursa @chouinar
-/analytics/ @ebuwa-evbuoma-fike @coilysiren @widal001 @AlexanderStephensonUSDS
+/documentation/ @mxk0  @widal001 @andycochran @acouch
+/frontend/ @andycochran @acouch @doug-s-nava 
+/api/ @chouinar
```

Comment:
@coilysiren can you throw me on here, or I can if you're ok with me pushing a commit to your branch.

Reply:
@mdragon yeah go for it!